app/services/populate_issues.rb

```diff
@@ -0,0 +1,30 @@
+class PopulateIssues
```

Comment:
I was debating that as well. The reason I'll often create a separate service class has to do with wanting to provide greater flexibility for the future. If there's a point in time (even now) when one might want to run this manually in the console, I like having a class I can access and use directly. That being said, I suppose there's no reason to not just do that within `PopulateIssuesJob`.